subprojects/docs/src/docs/userguide/javaTesting.adoc

```diff
@@ -15,8 +15,55 @@
 [[java_testing]]
 == Testing in Java & JVM projects
 
+////
```

Comment:
❗️ Did you mean to remove this?

Reply:
Remove what? Oh, are you asking whether I meant to remove the comment before committing? No, I want it left there. It's too hard for people working on the chapters to find the associated integration tests otherwise. Plus the command line to run those tests is too complex to remember. If I come back to this chapter, I definitely want that information there.